docs/fsharp/language-reference/discriminated-unions.md

```diff
@@ -81,6 +81,24 @@ let getShapeHeight shape =
 
 Normally, the case identifiers can be used without qualifying them with the name of the union. If you want the name to always be qualified with the name of the union, you can apply the [RequireQualifiedAccess](https://msdn.microsoft.com/library/8b9b6ade-0471-4413-ac5d-638cd0de5f15) attribute to the union type definition.
 
+### Unwrapping DU's
+
+Single case unions like this can be easily unwrapped without explicit matching.
```

Comment:
It's actually a form of pattern matching to unwrap like that.  I think you're saying that it's easy to unwrap single values without needing to use the `match` or `function` expressions, which is true.